packages/attribute-registry-process/src/routers/AttributeRouter.ts

```diff
@@ -134,21 +126,13 @@ const attributeRouter = (
       async (req, res) => {
         try {
           const { origin, code } = req.params;
-
           const attribute =
             await attributeRegistryService.getAttributeByOriginAndCode({
               origin,
               code,
             });
-          if (attribute) {
-            return res.status(200).json(toApiAttribute(attribute.data)).end();
-          } else {
-            const errorRes = makeApiProblem(
-              attributeNotFound(`${origin}/${code}`),
-              getAttributeByOriginAndCodeErrorMapper
-            );
-            return res.status(errorRes.status).json(errorRes).end();
-          }
```

Comment:
Refactor. There is no need to throw `attributeNotFound` inside the router. It can be thrown inside `attributeRegistryService`.